### ISSUE TITLE
parse_comparisons returning nothing when given OrderedDict

### ISSUE THREAD

**Issue by sallyrobson** [e4st-dev/E4ST.jl#217]

**Comment by sallyrobson** [comment]
the main issue here was wanting to specify gen_filters in the config like 
``` yaml
build_status: 
      - "built"
      - "unretrofitted"
```
which reads into the mod as 
``` julia
"build_status" => ["built", "unretrofitted"]
```

instead of specifying it like 
``` yaml 
build_status: "[built, unretrofitted]"
```
which reads into the mod as 
``` julia
"build_status" => "[built, unretrofitted]"
```

With how parse_comparisons() is formulated currently, we can only use the second option because there is only a method for parse_comparison() for an AbstractString instead of for a Vector of Strings. We could totally write a method for parse_comparison that works with a Vector but for now we are just requiring that you specify the gen_filters in the second format.

**Comment by Ethan-Russell** [comment]
Closed by #218
